ddtrace/tracer/sqlcomment_test.go

```diff
@@ -141,6 +141,27 @@ func TestSQLCommentCarrier(t *testing.T) {
 	}
 }
 
+func TestExtractOpenTelemetryTraceInformation(t *testing.T) {
+	spanID := generateSpanID(now())
+	traceID := generateSpanID(now())
+	priority := 1
+	// open-telemetry implementation appends comment to the end of the query
+	q := fmt.Sprintf("/*c*/ SELECT * from FOO /**/ /*dddbs='whiskey-db',dde='test-env',ddps='whiskey-service',ddpv='1.0.0',traceparent='00-0000000000000000%s-%s-0%d'*/", strconv.FormatUint(traceID, 16), strconv.FormatUint(spanID, 16), priority)
```

Comment:
This seems fine as a test case and can't find a fault with it as far as the functionality that it's testing but it might actually be misleading to future readers and make them think it's a typical example of an open-telemetry implementation. Here's an actual example from [the spec](https://google.github.io/sqlcommenter/spec/): 

```
SELECT * FROM FOO
/*action='%2Fparam*d',controller='index,'framework='spring',
traceparent='00-5bd66ef5095369c7b0d1f8f4bd33716a-c532cb4098ac3dd2-01',
tracestate='congo%3Dt61rcWkgMzE%2Crojo%3D00f067aa0ba902b7'*/
```

Reply:
Thanks for the suggestion. I updated the implementation to better handle these cases and keep the regex relatively simple.